PWGHF/TableProducer/candidateCreator2Prong.cxx

```diff
@@ -95,6 +95,7 @@ struct HfCandidateCreator2Prong {
   Service<o2::ccdb::BasicCCDBManager> ccdb;
 
   using TracksWCovExtraPidPiKa = soa::Join<aod::TracksWCovExtra, aod::TracksPidPi, aod::PidTpcTofFullPi, aod::TracksPidKa, aod::PidTpcTofFullKa>;
+  using BCs = soa::Join<aod::BCsWithTimestamps, aod::BcSels, aod::Run3MatchedToBCSparse>;
```

Comment:
How will this run on Run 2 data?

Reply:
I don't know, but from https://github.com/AliceO2Group/AliceO2/blob/8e0693280e134e41f0fe3d253bdda26b49831a80/Framework/Core/include/Framework/AnalysisDataModel.h#L2182-L2183, it miss FV0CId in run3

Reply:
There should be an option to process Run 2 data.

Reply:
> There should be an option to process Run 2 data.

I would not do that, since it will never be used from our side, and I didn't see it be used even in UD group.